tequilapi/endpoints/connection.go

```diff
@@ -98,18 +99,24 @@ type statisticsResponse struct {
 	Duration int `json:"duration"`
 }
 
+// SessionStatsKeeper represents the session stat keeper
+type SessionStatsKeeper interface {
+	Retrieve() consumer.SessionStatistics
+	GetSessionDuration() time.Duration
+}
+
 // ConnectionEndpoint struct represents /connection resource and it's subresources
 type ConnectionEndpoint struct {
 	manager     connection.Manager
 	ipResolver  ip.Resolver
-	statsKeeper stats.SessionStatsKeeper
+	statsKeeper SessionStatsKeeper
```

Comment:
We have renamed `statsKeeper` to `statisticsTracker` in a different place, should we do the same here too?

Reply:
renamed